### ISSUE TITLE
Show number of units left in the pool

### ISSUE THREAD

**Issue by formigao** [auto-chess-ui-mod/download#7]
Can you add info of how many units are left on the pool ?
% is nice but dont tell much. it's better to know let's say 11/45 left in pool

**Comment by TiesdeKok** [comment]
I agree that this would be nice to have in the list of heroes. The primary reason why I have not included it currently is to avoid it becoming too cluttered, there is already a lot of information in that list. 

But I will see if I can figure out a way to squish it in nicely.
